tests/test_util.py

```diff
@@ -27,3 +28,76 @@ def test_sph2cart(coord, polar):
     alpha, beta, r = polar
     b = sfs.util.sph2cart(alpha, beta, r)
     assert_allclose(b, coord)
+
+
+direction_vector_data = [
+    ((np.pi/4, np.pi/4), (0.5, 0.5, np.sqrt(2)/2)),
+    ((3*np.pi/4, 3*np.pi/4), (-1/2, 1/2, -np.sqrt(2)/2)),
+    ((3*np.pi/4, -3*np.pi/4), (1/2, -1/2, -np.sqrt(2)/2)),
+    ((np.pi/4, -np.pi/4), (-1/2, -1/2, np.sqrt(2)/2)),
+    ((-np.pi/4, np.pi/4), (1/2, -1/2, np.sqrt(2)/2)),
+    ((-3*np.pi/4, 3*np.pi/4), (-1/2, -1/2, -np.sqrt(2)/2)),
+    ((-3*np.pi/4, -3*np.pi/4), (1/2, 1/2, -np.sqrt(2)/2)),
+    ((-np.pi/4, -np.pi/4), (-1/2, 1/2, np.sqrt(2)/2)),
+]
+
+
+@pytest.mark.parametrize('input, vector', direction_vector_data)
+def test_direction_vector(input, vector):
+    alpha, beta = input
+    c = sfs.util.direction_vector(alpha, beta)
+    assert_allclose(c, vector)
+
+
+db_data = [
+    (0, -np.inf),
+    (1, 0),
+    (10, 10),
+    (10*2, (10+3.010299956639813)),
+    (10*10, (10+10)),
+    (10*3, (10+4.771212547196624)),
+    (10*4, (10+6.02059991327962)),
+    (10*0.5, (10-3.01029995663981198)),
+    (10*0.1, (10-10)),
+    (10*0.25, (10-6.02059991327962396))
+    ]
+
+
+@pytest.mark.parametrize('linear, decibel', db_data)
+def test_db_amplitude(linear, decibel):
+    d = sfs.util.db(linear, True)
```

Comment:
This doesn't match the function name.

Also, you should not use a bare `True` or `False` as a function argument (unless probably if it's the only argument).
You should use it as a keyword argument, i.e. you should use `power=True` (except you should actually use that in `test_db_power()`).